throw new Error('Outer error.');